jaggeryapps/dashboard/directories/delete_finish.jag

```diff
@@ -0,0 +1,46 @@
+<%
+
+var log = new Log();
+include("../controllers/wsUtil.jag");
+var util = require("../util/utility.jag");
+
+var serverUrl = util.getServerUrl();
+serverUrl = serverUrl + "/UserStoreConfigAdminService";
+var ws = require('ws');
+var webService = new ws.WSRequest();
+var i18n = session.get("i18n");
```

Comment:
var i18n = session.get("i18n"); No usages in any places... is that expected to use it later